keyboards/handwired/practice60/rules.mk

```
@@ -0,0 +1,13 @@
+#BOOTMAGIC_ENABLE = yes	# Virtual DIP switch configuration
+MOUSEKEY_ENABLE = yes	# Mouse keys
+EXTRAKEY_ENABLE = yes	# Audio control and System control
+CONSOLE_ENABLE = yes	# Console for debug
+COMMAND_ENABLE = yes    # Commands for debug and configuration
+SLEEP_LED_ENABLE = yes  # Breathing sleep LED during USB suspend
+NKRO_ENABLE = yes	    # USB Nkey Rollover
+CUSTOM_MATRIX = yes # Custom matrix file
+BACKLIGHT_ENABLE = yes
+
+LAYOUTS = 60_ansi
+
+DEFAULT_FOLDER = practice60/bluepill
```

Ithink this needs to be: 
```suggestion
DEFAULT_FOLDER = handwired/practice60/bluepill
```